src/psyclone/transformations.py

```diff
@@ -378,6 +378,8 @@ def validate_it_can_run_on_gpu(self, node, options):
         :type options: Optional[Dict[str, Any]]
         :param bool options["force"]: whether to allow routines with
             CodeBlocks to run on the GPU.
+        :param bool options["device_string"]: provide a compiler-platform
+            identifier strign.
```

Comment:
"identifier." and s/bool/str/

Reply:
Please could you also describe this new option in the class doc string in order to publicise it.